### ISSUE TITLE
Data model validation error: 7 validation errors for HistoricalWhois

### ISSUE THREAD

**Issue by Nealtron** [pirxthepilot/wtfis#1]
Experiencing whois error when querying specific domain (some of the sub-domains are also affected):

```
$ wtfis mymanatee.org
✓ Fetching data from Virustotal         ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
✓ Fetching IP enrichments from IPWhois  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
⠙ Fetching domain whois from Virustotal ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% 0:00:00
Data model validation error: 7 validation errors for HistoricalWhois
data -> 6 -> attributes -> whois_map -> Domain Name
  field required (type=value_error.missing)
data -> 6 -> attributes -> whois_map -> Registrar
  field required (type=value_error.missing)
data -> 6 -> attributes -> whois_map -> Name Server
  field required (type=value_error.missing)
data -> 6 -> attributes -> whois_map -> Creation Date
  field required (type=value_error.missing)
data -> 6 -> attributes -> whois_map -> Registry Expiry Date
  field required (type=value_error.missing)
data -> 6 -> attributes -> whois_map -> Updated Date
  field required (type=value_error.missing)
data -> 6 -> attributes -> last_updated
  field required (type=value_error.missing)
```

**Comment by pirxthepilot** [comment]
Hi @Nealtron, thanks for the report! I will take a look. Would you be able to share the domain you are querying that is causing the error? (It's okay if you can't!)

**Comment by Nealtron** [comment]
@pirxthepilot, the domain is "mymanatee.org".

**Comment by pirxthepilot** [comment]
@Nealtron just published v0.0.5 to pypi - Please update and test again when you get the chance. Thanks!
